Sources/Core/GTMSessionFetcher.m

```diff
@@ -2991,6 +3000,14 @@ - (void)URLSession:(NSURLSession *)session
     }
   }  // @synchronized(self)
 
+  if (self.stopFetchingTriggersCallbacks && self.userStoppedFetching) {
+    NSError *error = [NSError errorWithDomain:kGTMSessionFetcherErrorDomain
+                                         code:GTMSessionFetcherErrorCancelled
+                                     userInfo:@{@"description" : @"Operation cancelled"}];
```

Comment:
Would we want to also add NSUnderlyingErrorKey with the cancellation error from NSURLSession? Not sure whether I think so, but wanted to ask the question.